src/AssemblyAI/Core/AssemblyAIClientApiException.cs

```diff
@@ -0,0 +1,27 @@
+using AssemblyAI.Core;
+
+#nullable enable
+
+namespace AssemblyAI.Core;
```

Comment:
We don't want to expose a `Core` namespace, so let's put this at the root namespace.